Documentation/platforms/arm/rp2040/boards/seeed-xiao-rp2040/README.txt

```diff
@@ -0,0 +1,96 @@
+README
+======
+
+This directory contains the port of NuttX to the Seeed Studio Xiao RP2040.
+See ttps://wiki.seeedstudio.com/XIAO-RP2040/ for information about Seeed
+Studio Xiao RP2040.
+
+NuttX supports the following RP2040 capabilities:
+  - UART  (console port)
+    - GPIO 0 (UART0 TX) and GPIO 1 (UART0 RX) are used for the console.
+  - I2C
+  - SPI (master only)
+  - DMAC
+  - PWM
+  - ADC
+  - Watchdog
+  - USB device
+    - MSC, CDC/ACM serial and these composite device are supported.
+    - CDC/ACM serial device can be used for the console.
+  - PIO (RP2040 Programmable I/O)
+  - Flash ROM Boot
+  - SRAM Boot
+    - If Pico SDK is available, nuttx.uf2 file which can be used in
+      BOOTSEL mode will be created.
+  - Persistent flash filesystem in unused flash ROM
+
+NuttX also provide support for these external devices:
```

Comment:
```suggestion
NuttX also provides support for these external devices:
```

Reply:
Hi @hartmannathan ,
It has been fixed.
Thanks